pdc_client/plugins/release_variant.py

```diff
@@ -52,14 +52,17 @@ def register(self):
         delete_parser.add_argument('uid', nargs="+")
         delete_parser.set_defaults(func=self.release_variant_delete)
 
-    def add_release_variant_arguments(self, parser, required=False):
+    def add_release_variant_arguments(self, parser, required=False, exclude=False):
         required_args = OrderedDict()
         required_args["release"] = {}
         required_args["uid"] = {}
         required_args["id"] = {}
         required_args["name"] = {}
         required_args["type"] = {}
         required_args["arch"] = {"action": "append", "dest": "arches"}
+        if exclude:
+            required_args.pop("arch")
```

Comment:
I was thinking about maybe some parameters also need to pop in the future, so just use the exclude here.
Anyway, use exclude_arches to instead now.